defi/src/protocols/data3.ts

```diff
@@ -18982,7 +18982,7 @@ const data3: Protocol[] = [
     module: "shoebillFinance-v2/index.js",
     oraclesByChain: {
       wemix: ["Pyth"], // https://github.com/DefiLlama/defillama-server/pull/5434#issuecomment-1889510580
-      manta: ["Supra"], // https://twitter.com/ShoebillFinance/status/1771149332689166521
+      manta: ["Supra","RedStone"], // https://twitter.com/ShoebillFinance/status/1771149332689166521 https://twitter.com/redstone_defi/status/1763580020751245496
```

Comment:
Who is securing STONE price? Cant be both

Reply:
primary feeds are secured by redstone oracle, and we use supra secondary feeds with max deviation check

Reply:
primary feeds are secured by redstone oracle, and we use supra secondary feeds with max deviation check